weaviate/templates/weaviateSecretOpenAI.yaml

```diff
@@ -1,9 +1,13 @@
-{{ if and (index .Values "modules" "text2vec-openai" "enabled") (index .Values "modules" "text2vec-openai" "apiKey") }}
+{{ if or (and (index .Values "modules" "text2vec-openai" "enabled") (index .Values "modules" "text2vec-openai" "apiKey")) (and (index .Values "modules" "qna-openai" "enabled") (index .Values "modules" "qna-openai" "apiKey")) }}
```

Comment:
This is quite difficult to read, could we split this up into parts using variables? I think something like the following should be possible?

```
$t2vOpenAI := and (index .Values "modules" "text2vec-openai" "enabled") (index .Values "modules" "text2vec-openai" "apiKey")
$qnaOpenAI := and (index .Values "modules" "qna-openai" "enabled") (index .Values "modules" "qna-openai" "apiKey")
$openAI := or ($t2vOpenAI) (qnaOpenAI)
```

Haven't validated if this is actual syntax, but I think something along those lines should be possible.

Reply:
I'll try this out